index.html

```diff
@@ -13,47 +13,54 @@
   </head>
   <body>
     <!-- This is the nav code -->
-    <header>
-      <nav>
-        <a href=""><img src="images/Logo Placeholder.svg" alt="" /></a>
 
-        <a href=""><img src="images/Icon - Menu.svg" alt="" /></a>
-      </nav>
-    </header>
-    <div class="header-section">
-      <div>
-        <h1>
-          Hey there. I’m Joseph <br />
-          <span class="header-span"> I’m a software developer</span>
-        </h1>
-        <p>
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
-      </div>
-      <div class="social">
-        <a href="https://mobile.twitter.com/I_amBabakay"
-          ><img class="social-icons" src="images/twitter.svg" alt=""
-        /></a>
-        <a href="https://www.linkedin.com/in/olatunji-kayode/"
-          ><img class="social-icons" src="images/Linkedin icon.svg" alt=""
-        /></a>
-        <a href="https://medium.com/@kayodej4"
-          ><img class="social-icons" src="images/medium.svg" alt=""
-        /></a>
-        <a href="https://github.com/Jolak5"
-          ><img class="social-icons" src="images/github.svg" alt=""
-        /></a>
-        <a href="https://angel.co/u/kayode-joseph-olatunji"
-          ><img class="social-icons" src="images/angellist.svg" alt=""
-        /></a>
-      </div>
-      <div class="scroll-button">
-        <a href=""><img src="images/Disabled.svg" alt="" /></a>
-      </div>
-    </div>
+    <section class="desktop-header">
+      <header>
+        <nav class="desktop-nav">
+          <a href="">Portfolio</a>
+          <a href="">About</a>
+          <a href="">Contact</a>
+        </nav>
+        <nav class="mobile-nav">
+          <a href=""><img src="images/Logo Placeholder.svg" alt="" /></a>
 
+          <a href=""><img src="images/Icon - Menu.svg" alt="" /></a>
+        </nav>
+      </header>
+      <div class="header-section">
+        <div>
+          <h1>
+            Hey there. I’m Joseph <br />
+            <span class="header-span"> I’m a software developer</span>
+          </h1>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
```

Comment:
- [ ] Great job on making this look awesome, however, it's looking huge in the mobile version. Kindly decrease the `font-size` in mobile view 👍

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206260852-1b1252cd-b762-4a09-8e6c-c262e37e3b0b.png)

## Figma design:
 
![image](https://user-images.githubusercontent.com/50721479/206260946-7cb9b0db-9b4c-46a5-a5d3-97bc4d74c131.png)